modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor4/BaseEditor.js

```diff
@@ -102,6 +102,10 @@ const BaseEditor = forwardRef(
 				}
 
 				data = data.replace(/(\u200B){7}/, '');
+
+				if (editor.config?.extraPlugins.search(/bbcode|creole/) > -1) {
```

Comment:
Is there a reason why this is inside `if (editor && editor.instanceReady)`? What happens if bbcode/creol instance is not ready? Will `createElementFromHTML` create a div that still looks broken?

Reply:
I suspect if not wait until `editor.instanceReady` the `editor.config` can change because I see some methods to configure it in client extensions. But If you think it's better to move it outside `if (editor && editor.instanceReady)` I'm fine I can do it 😄

Reply:
@boton Yeah, please make this small change, and we're good to forward after.